enterprise/admin/views.py

```diff
@@ -322,13 +323,19 @@ def post(self, request, customer_uuid):
 
         # _handle_form might add form errors, so we check if it is still valid
         if manage_learners_form.is_valid():
-
-            # Enroll linked users in a course if requested
+            course_ids = []
             course_details = manage_learners_form.cleaned_data.get(ManageLearnersForm.Fields.COURSE)
+            program_details = manage_learners_form.cleaned_data.get(ManageLearnersForm.Fields.PROGRAM)
+
             if course_details:
+                course_ids.append(course_details['course_id'])
+            elif program_details:
```

Comment:
Is it ever possible for both course_details and program_details to exist simultaneously?